### PR TITLE
new version update

### PR DESCRIPTION
**Pull request by ankitdas13**
Doc : https://docs.google.com/document/d/1k9eJ9TmAi-2-ITxgukn2SbQWT5Bflpm77bOXWPHTTlk/edit?usp=sharing

### CODE REVIEW

**Comment by sonyy-m** [comment]
- Update requests library version in razorpay.php
- Rename tests file names to Pascal case (first letter should be uppercase)

**Comment by sonyy-m** [comment]
- Update requests library version in razorpay.php
- Rename tests file names to Pascal case (first letter should be uppercase)

**Comment by sonyy-m** [comment]
`Razorpay\Tests\QrCodeTest::testCloseQrCode` testcase failed due to undefined variable $customer. Add test customer_id
https://github.com/razorpay/razorpay-php/blob/b7c94db5044e9c667ec1b974573262a1be730b51/tests/qrCodeTest.php#L84-L88

**Comment by sonyy-m** [comment]
`Razorpay\Tests\QrCodeTest::testCloseQrCode` testcase failed due to undefined variable $customer. Add test customer_id
https://github.com/razorpay/razorpay-php/blob/b7c94db5044e9c667ec1b974573262a1be730b51/tests/qrCodeTest.php#L84-L88

**Review by sonyy-m — CHANGES_REQUESTED** [review]
Can we add test payment id in place of `$downtime['items'][0]['id']` to avoid undefined array key 0 error ?
https://github.com/razorpay/razorpay-php/blob/b7c94db5044e9c667ec1b974573262a1be730b51/tests/paymentTest.php#L103-L107

File: tests/TestCase.php [hunk 1]
```diff
@@ -4,14 +4,15 @@
...
     
-    public function setUp()
+    public function setUp(): void
     {
-        $apiKey = "api_key";
-        $apiSecret = "api_secret";
+        $apiKey = "rzp_test_1DP5mmOlF5G5ag";
+        $apiSecret = "thisissupersecret";
         
         $this->api = new Api( $apiKey, $apiSecret);
```
Comment (sonyy-m) [inline tests/TestCase.php:17]: Remove Api keys

Comment (sonyy-m) [inline tests/TestCase.php:17]: Remove Api keys

Comment (ankitdas13) [reply tests/TestCase.php]: removed

**Review by sonyy-m — APPROVED** [review]
LGTM

**Pull request merged**